docs/distribution/README.md

```diff
@@ -0,0 +1,923 @@
+          }
+        }
+      ]
+    }
+    ```
+
+    The `manifest.config` contains the following signature information:
+
+    ```json
+    {
+        "signed": {
+            "mediaType": "application/vnd.oci.image.manifest.v2+json",
+            "digest": "sha256:111ma2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49a111m",
+            "size": 528,
+            "references": [
+                "registry.wabbit-networks.com/net-monitor:v1"
+            ],
+            "exp": 1627555319,
+            "nbf": 1596019319,
+            "iat": 1596019319
+        },
+        "signature": {
+            "typ": "x509",
+            "sig": "UFqN24K2fLj...",
+            "alg": "RS256",
+            "x5c": [
+                "MIIDszCCApugAwIBAgIUL1anEU/..."
+            ]
+        }
+    }
+    ```
+
+3. **index digest for the acme-rockets signature** `sha256:333ic0c33ebc4a74a0a554c86ac2b28ddf3454a5ad9cf90ea8cea9f9e75c333i`
+
+    ```json
+    {
+      "schemaVersion": 2,
+      "mediaType": "application/vnd.oci.image.index.v2+json",
+      "config": {
+        "mediaType": "application/vnd.cncf.notary.config.v2+jwt",
+        "digest": "sha256:333cc44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b785c333c",
+        "size": 1906
+      },
+      "manifests": [
+        {
+          "mediaType": "application/vnd.oci.image.manifest.v1+json",
+          "digest": "sha256:111ma2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49a111m",
+          "size": 7023,
+          "platform": {
+            "architecture": "ppc64le",
+            "os": "linux"
+          }
+        }
+      ]
+    }
+    ```
+
+    The `manifest.config` contains the following signature information:
+
+    ```json
+    {
+        "signed": {
+            "mediaType": "application/vnd.oci.image.manifest.v2+json",
+            "digest": "sha256:111ma2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49a111m",
+            "size": 528,
+            "references": [
+                "registry.acme-rockets.com/net-monitor:v1"
+            ],
+            "exp": 1627555319,
+            "nbf": 1596019319,
+            "iat": 1596019319
+        },
+        "signature": {
+            "typ": "x509",
+            "sig": "UFqN24K2fLj...",
+            "alg": "RS256",
+            "x5c": [
+                "MIIDszCCApugAwIBAgIUL1anEU/..."
+            ]
+        }
+    }
+    ```
+
+**Pros with this approach:**
+
+* Utilize the existing `index.manifests` collection for linking artifacts
+* Registries that support oci index already have infrastructure for tracking `index.manifests`, including delete operations and garbage collection
+* Existing distribution-spec upload APIs are utilized
+* Unlike the manifest proposal, no additional artifact handler would be required to parse the config object for linking artifacts
+* Based on the artifact type:  `manifest.config.mediaType: "application/vnd.cncf.notary.config.v2+jwt"`, role check may be done to confirm the identity has a signer role
```

Comment:
Steve, let me clarify:
Is this signer role optionally required based on the implementation of the registry service, or part of any spec?

Reply:
That's a great question. The distribution spec doesn't call out roles at all.  Yet, all registries support a set of roles. 
What roles should a signature capability call out as a requirement in the spec, vs. a capability we know registry operators will implement and how do we acknowledge that need without putting it in the spec?

With the recent feedback that registries may want to assign roles to different artifacts, I'm inclined to think of this as nothing specific to signatures and not actually call out a signing role as a spec requirement.

Reply:
I've added sweet #16 to capture the question of roles.